tests/test_css.py

```diff
@@ -110,40 +110,39 @@ def test_valid_css():
             '<p style="background: #00D;">foo</p>',
             '<p style="background: #00D;">foo</p>',
         ),
-        # Verify urls with no quotes, single quotes, and double quotes are all dropped
         (
             '<p style="background: url(topbanner.png) #00D;">foo</p>',
-            '<p style="background: #00D;">foo</p>',
+            '<p style="background: url(topbanner.png) #00D;">foo</p>',
         ),
         (
             "<p style=\"background: url('topbanner.png') #00D;\">foo</p>",
-            '<p style="background: #00D;">foo</p>',
+            "<p style='background: url(\"topbanner.png\") #00D;'>foo</p>",
         ),
         (
             "<p style='background: url(\"topbanner.png\") #00D;'>foo</p>",
-            '<p style="background: #00D;">foo</p>',
+            "<p style='background: url(\"topbanner.png\") #00D;'>foo</p>",
         ),
         # Verify urls with spacing
         (
             "<p style=\"background: url(  'topbanner.png') #00D;\">foo</p>",
-            '<p style="background: #00D;">foo</p>',
+            "<p style='background: url(  \"topbanner.png\") #00D;'>foo</p>",
         ),
         (
             "<p style=\"background: url('topbanner.png'  ) #00D;\">foo</p>",
-            '<p style="background: #00D;">foo</p>',
+            "<p style='background: url(\"topbanner.png\"  ) #00D;'>foo</p>",
         ),
         (
             "<p style=\"background: url(  'topbanner.png'  ) #00D;\">foo</p>",
-            '<p style="background: #00D;">foo</p>',
+            "<p style='background: url(  \"topbanner.png\"  ) #00D;'>foo</p>",
         ),
         (
             "<p style=\"background: url (  'topbanner.png'  ) #00D;\">foo</p>",
-            '<p style="background: #00D;">foo</p>',
+            "<p style='background: url (  \"topbanner.png\"  ) #00D;'>foo</p>",
```

Comment:
I'm not sure we need all these variations anymore, but I didn't spend the time to think through what they're actually testing with the new code.